infra/scripts/test_ubuntu_npud.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eo pipefail
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_PATH="$(cd ${CURRENT_PATH}/../../ && pwd)"
+
+# Install path on CI
+INSTALL_PATH="$ROOT_PATH/Product/out"
+
+# Install dbus configuration file
+DBUS_CONF="${INSTALL_PATH}/share/org.tizen.npud.conf"
+mkdir -p /usr/share/dbus-1/system.d/
+cp ${DBUS_CONF} /usr/share/dbus-1/system.d/
+
+function NpudTest()
+{
+  pushd ${ROOT_PATH} > /dev/null
+
+  $INSTALL_PATH/unittest/npud_test
```

Comment:
If we need additional setting for `npud_test`, it's better to install `npud_test` to different directory (like `nnapi_gtest`).

ex) `npud-test/npud_test`

Unittests in `unittest/` are tested on CI by `$INSTALL_PATH/test/onert-test unittest` command automatically.